web/packages/design/src/ThemeProvider/index.tsx

```diff
@@ -59,18 +60,25 @@ const ThemeProvider = props => {
     };
   }, [themePreference]);
 
-  return (
-    <StyledThemeProvider
-      theme={props.theme || themePreferenceToTheme(themePreference)}
-    >
-      <StyleSheetManager disableVendorPrefixes>
-        <React.Fragment>
-          <GlobalStyle />
-          {props.children}
-        </React.Fragment>
-      </StyleSheetManager>
-    </StyledThemeProvider>
-  );
+  // If customTheme is set, force that to be the theme regardless of user preferences.
+  let theme;
+  if (cfg.customTheme === 'bblp') {
```

Comment:
This is okay, but we have to remember to update it every time a new custom theme is added.

If instead we had something like:

```
customThemes = {
    'bblp': {  ... },
    'other': { ... },
}

if (customThemes.hasOwnProperty(cfg.customTheme)) {
   // use custom theme
} else {
   // use preference
}
```